queries.sql

```diff
@@ -208,4 +208,68 @@ UPDATE
 SET
     weigth_kg = weigth_kg *(-1);
 
-commit;
\ No newline at end of file
+commit;
+
+SELECT
+    a.name
+FROM
+    animals a
+    JOIN owners v ON a.owner_id = v.id
+WHERE
+    v.full_name = 'Melody';
```

Comment:
- [ ] Same here as pointed out in the previous comment :point_up_2:  Kindly use the full name.